Pin numpy to 1.26.x

oursland opened the pull request:
Currently `numpy>=2` have issues when installed from conda-forge.  Furthermore, VFX Platform currently standardizes on `numpy==1.26`.

This PR pins numpy to 1.26 which should resolve the current issues plaguing recent releases and ensuring compatibility with other VFX Platform compliant systems.

oursland commented:
Addresses #375.

hyarion commented:
Should we document why we pin to these versions or how do we update them safely in the future?

oursland commented:
I've merged this to get it in before the next weekly build.  I'll test it after the build completes to see if it addresses the issues experienced by macros and workbenches that depend upon `numpy`.
